kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -212,4 +309,33 @@
   .sign-in-error
     color: red
 
+  .suggestions
```

Comment:
These classes come from the KeenUI UiAutocomplete component - because of lack of Aria support and for consistency, I just took the suggestion list part, and not the whole component, so that we could use our preexisting textbox.